validator/src/main.rs

```diff
@@ -2516,7 +2437,11 @@ pub fn main() {
     });
 
     if !matches.is_present("no_os_network_limits_test") {
-        check_os_network_limits();
+        if SystemMonitorService::check_os_network_limits() {
+            info!("OS network limits test passed.");
+        } else {
+            eprintln!("OS network limit test failed. solana-sys-tuner may be used to configure OS network limits. Bypass check with --no-os-network-limits-test.");
```

Comment:
why eprintln instead of warn?

Reply:
I want to add `exit(1)` here, but was going to do that with a separate commit so this change could be backported independent of `exit(1)`. Change to `eprintln` was to match the above checks which fail with `exit(1)`. Thoughts?

Reply:
Sorry I'm not quite following. I can understand not adding an exit(1) so this can be backported.
which 'above checks' fail with exit(1)?
I'm sure a quick call wed will clear this up.

Reply:
Sorry, I was just referring to other occurrences of sanity checking in this file that call `eprintln ... exit(1)` when validation fails. For example: `--limit-ledger-size` [validation](https://github.com/solana-labs/solana/blob/e616a7ebfc16f88abf66a0329a9ff0d526a46a6a/validator/src/main.rs#L2498) on line 2497. This was just for consistency with the surrounding code.

Reply:
sure enough. looks ok to me. I had not seen eprintln yet.